src/librustc_mir/hair/pattern/mod.rs

```diff
@@ -175,6 +175,11 @@ pub enum PatternKind<'tcx> {
         slice: Option<Pattern<'tcx>>,
         suffix: Vec<Pattern<'tcx>>,
     },
+
+    /// or-pattern
```

Comment:
```suggestion
    /// An or-pattern, e.g. `p | q`.
    /// Invariant: `pats.len() >= 2`.
```
Seems the suggestion from before was lost.